src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -927,6 +927,34 @@ namespace winrt::TerminalApp::implementation
         _ShowAboutDialog();
     }
 
+    // Method Description:
+    // Called when the users pressed keyBindings while CommandPalette is open.
+    // Arguments:
+    // - e: the KeyRoutedEventArgs containing info about the keystroke.
+    // Return Value:
+    // - <none>
+    void TerminalPage::_CommandPaletteKeyDownHandler(Windows::Foundation::IInspectable const& /*sender*/, Windows::UI::Xaml::Input::KeyRoutedEventArgs const& e)
```

Comment:
Consider renaming this into `_KeyDownHandler` because it now serves the `TerminalPage` in general: it runs the bindings no matter where you are. It does have a specific logic for CommandPalette, but tomorrow we will probably want to do more stuff.